app/Http/Controllers/API/VoucherController.php

```diff
@@ -260,10 +283,11 @@ private function constructResponseMessage($responses)
                 'message' => trans(
                     'api.messages.batch_voucher_submit',
                     [
-                    'success_amount' => count($responses['success_add']),
-                    'duplicate_amount' => count($responses['own_duplicate'])
-                        + count($responses['other_duplicate']),
-                    'invalid_amount' => count($responses['invalid']),
+                        'success_amount' => count($responses['success_add']),
+                        'duplicate_amount' => count($responses['own_duplicate'])
+                            + count($responses['other_duplicate']),
```

Comment:
I know this is how it was before, but it feels like other duplicate should be counted as invalid.